triton-inference-client.spec

```diff
@@ -44,25 +47,46 @@ cmake ${PROJ_DIR} \
     -DCMAKE_CXX_STANDARD=%{cms_cxx_standard} \
     -DTRITON_ENABLE_CC_HTTP=OFF \
     -DTRITON_ENABLE_CC_GRPC=ON \
-    -DTRITON_ENABLE_PYTHON_HTTP=OFF \
-    -DTRITON_ENABLE_PYTHON_GRPC=OFF \
     -DTRITON_ENABLE_PERF_ANALYZER=OFF \
     -DTRITON_ENABLE_EXAMPLES=OFF \
     -DTRITON_ENABLE_TESTS=OFF \
     -DTRITON_USE_THIRD_PARTY=OFF \
     -DTRITON_KEEP_TYPEINFO=ON \
     -DTRITON_COMMON_REPO_TAG=${common_tag} \
     -DTRITON_ENABLE_GPU=${TRITON_ENABLE_GPU_VALUE} \
+    -DCMAKE_CXX_FLAGS="-Wno-error -fPIC" \
+    -DFETCHCONTENT_SOURCE_DIR_REPO-COMMON=${COMMON_DIR} \
+    -DCMAKE_PREFIX_PATH="${GRPC_ROOT};${ABSEIL_CPP_ROOT};${RE2_ROOT};${RAPIDJSON_ROOT}"
+
+make %{makeprocesses} VERBOSE=1
+
+rm -rf ../buildpy ; mkdir ../buildpy ; cd ../buildpy
+cmake ../%{n}-%{realversion}/src/python \
+    -DCMAKE_INSTALL_PREFIX="%{i}" \
+    -DCMAKE_INSTALL_LIBDIR=lib \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_CXX_STANDARD=%{cms_cxx_standard} \
+    -DTRITON_ENABLE_PYTHON_HTTP=OFF \
```

Comment:
Following the above comment, I would include the corresponding lines here:
```
    -DTRITON_ENABLE_CC_HTTP=OFF \
    -DTRITON_ENABLE_CC_GRPC=OFF \
```

Reply:
these flags are not available for `triton/src/python/CMakeLists.txt` which we use to build the python interface